modules/contact/test/tests/explicit_dynamics/tests

```diff
@@ -21,39 +21,16 @@
     requirement = 'The system shall be able to solve a simple few-element normal contact problem '
                   'using explicit dynamics solving uncoupled, local equations of momentum balance.'
   []
-  [test_balance_overwrite]
-    type = 'CSVDiff'
-    input = 'test_balance.i'
-    csvdiff = 'test_balance_overwrite_out.csv'
-    cli_args = 'ExplicitDynamicsContact/my_contact/overwrite_current_solution=true Outputs/file_base=test_balance_overwrite_out'
-    abs_zero = 1.0e-4
-    allow_warnings = true
-    method = '!dbg'                  
-    requirement = 'The system shall be able to solve a simple few-element normal contact problem '
-                  'using explicit dynamics solving uncoupled, local equations of momentum balance '
-                  ' and overwrite boundary variables after applying the time stepper scheme.'
-  []
   [test_balance_short]
     type = 'Exodiff'
     input = 'test_balance.i'
     exodiff = 'test_balance_short_out.e'
     abs_zero = 1.0e-4
     allow_warnings = true
-    cli_args = 'Outputs/file_base=test_balance_short_out Executioner/end_time=-0.009'            
+    cli_args = 'Outputs/file_base=test_balance_short_out Executioner/end_time=0.001'            
     requirement = 'The system shall be able to solve a simple few-element normal contact problem '
                   'using explicit dynamics solving uncoupled, local equations of momentum balance in debug mode.'
   []
-  [test_balance_overwrite_short]
-    type = 'CSVDiff'
-    input = 'test_balance.i'
-    csvdiff = 'test_balance_overwrite_short_out.csv'
-    cli_args = 'ExplicitDynamicsContact/my_contact/overwrite_current_solution=true Outputs/file_base=test_balance_overwrite_short_out Executioner/end_time=-0.009'
```

Comment:
If you are removing these tests, remove gold and input files as well.

Reply:
@maxnezdyur will address this in a follow on PR